Makefile

```diff
@@ -1,7 +1,7 @@
 ADB = ${ANDROID_HOME}/platform-tools/adb
 GRADLE = ./gradlew
 GRADLE_OPTS = --daemon --parallel
-flavor = UnbrandedWebview
```

Comment:
Please rollback this change, you can still build the flavor from the command line passing your flavor as argument, eg.:

```make flavor=alerte_nigerWebView assemble```

Reply:
rolled back